webapp/src/app/api/v0/cas/login/login.component.ts

```diff
@@ -1,4 +1,4 @@
-import crypto from "crypto-browserify";
+import crypto from "crypto";
```

Comment:
Please use `crypto-browserify` for the moment. Else we have two different sources of crypto, and I don't want to chase those bugs.